src/engine/SCons/Tool/install.xml

```diff
@@ -7,129 +7,198 @@ See its __doc__ string for a discussion of the format.
-(the directory that contains the &SConstruct; file,
-usually the current directory).
-Use command line targets or the &Default; function
-in this case.
-</para>
-
-<para>
-If the <option>--install-sandbox</option> command line
-option is given, the target directory will be prefixed
-by the directory path specified.
-This is useful to test installs without installing to
-a "live" location in the system.
-</para>
-
-<para>
-See also &FindInstalledFiles;.
-For more thoughts on installation, see the User Guide
-(particularly the section on Command-Line Targets
-and the chapters on Installing Files and on Alias Targets).
-</para>
-
-</summary>
-</builder>
-
-<builder name="InstallAs">
-<summary>
-<para>
-Installs one or more source files or directories
-to specific names,
-allowing changing a file or directory name
-as part of the installation.
-It is an error if the
-target
-and
-source
-arguments list different numbers of files or directories.
-</para>
-
-<example_commands>
-env.InstallAs(target='/usr/local/bin/foo',
-              source='foo_debug')
-env.InstallAs(target=['../lib/libfoo.a', '../lib/libbar.a'],
-              source=['libFOO.a', 'libBAR.a'])
-</example_commands>
-
-<para>
-See the note under &Install;.
-</para>
-
-</summary>
-</builder>
-
-<builder name="InstallVersionedLib">
-<summary>
-<para>
-Installs a versioned shared library. The symlinks appropriate to the
-architecture will be generated based on symlinks of the source library.
-</para>
-
-<example_commands>
-env.InstallVersionedLib(target='/usr/local/bin/foo',
-                        source='libxyz.1.5.2.so')
-</example_commands>
-
-<para>
-See the note under &Install;.
-</para>
-
-</summary>
-</builder>
+    <tool name="install">
+        <summary>
+            <para>
+                Sets construction variables for file
+                and directory installation.
+            </para>
+        </summary>
+        <sets>
+            <item>INSTALL</item>
+            <item>INSTALLSTR</item>
+            <item>INSTALLFILECOPY</item>
+            <item>INSTALLFILECOPYFLAGS</item>
+            <item>INSTALLDIRCOPY</item>
+            <item>INSTALLDIRCOPYFLAGS</item>
+        </sets>
+    </tool>
+
+    <cvar name="INSTALLFILECOPY">
+        <summary>
+            <para>
+                The command to call to copy a file as part of the Install() and InstallAs() builders.
+            </para>
+            <para>
+                For Posix systems defaults to
+                <example_commands>cp</example_commands>
+            </para>
+            <para>
+                For Win32 systems defaults to
+                <example_commands>Xcopy</example_commands>
+            </para>
+        </summary>
+    </cvar>
+
+    <cvar name="INSTALLFILECOPYFLAGS">
+        <summary>
+            <para>
+                Flags to add to the command line for copying a file as part of the Install() and InstallAs() builders.
+            </para>
+            <para>
```

Comment:
Should there be an explanation of why these flags were chosen? (i.e. to preserve ownership and permission modes)

Reply:
Sounds good. Adding.